app/views/catalog/_home_text.html.erb

```diff
@@ -40,4 +40,7 @@
       <%= content_tag :div, '', id: 'map', aria: { label: t('geoblacklight.map.label') }, data: { map: 'home', 'catalog-path'=> search_catalog_path , 'map-bbox' => params[:bbox], basemap: geoblacklight_basemap, leaflet_options: leaflet_options } %>
     </div>
   </div>
+  <div align='right'>
```

Comment:
I think we might want to take another look at how / where this is styled. Are we trying to put this within the visible column/row layout?